tensorflow/python/keras/mixed_precision/layer_correctness_test.py

```diff
@@ -139,6 +139,11 @@ def _create_model_from_layer(self, layer, input_shapes):
        (2, 2, 2)),
       ('Bidirectional',
        lambda: wrappers.Bidirectional(recurrent.SimpleRNN(units=4)), (2, 2, 2)),
+      ('AttentionLayer', dense_attention.Attention,
```

Comment:
Pass `causal=True` here, as in the next test. Since there already is an `Attention` test above that runs without causal.

Reply:
Thanks @reedwm, updated.